elements/rh-demo/rh-demo.ts

```diff
@@ -0,0 +1,63 @@
+import { LitElement, html } from 'lit';
+import { customElement } from 'lit/decorators.js';
+
+import style from './rh-demo.css';
+
+@customElement('rh-demo')
+export class RhDemo extends LitElement {
+  static readonly styles = style;
+
+  constructor() {
+    super();
+    import('@patternfly/pfe-accordion');
```

Comment:
@bennypowers to your point about two different pfe-accordions.  This line is mostly leftover code from the original POC.  My thought was to get rid of it and have pfe-accordion a "requirement" of using this component.  That's my get out of jail free card option.

Reply:
That makes sense but we'd probably still want to import accordion here.


I presented 3 options in the systems feds chat, will see what they say